sass/cd/cd-header/_cd-user-menu.scss

```diff
@@ -215,7 +207,46 @@
   content: none;
 }
 
+// Last block in Global Header shouldn't have a separator or padding */
+// .cd-global-header__user-menu:last-child .menu > li > a,
+// .cd-global-header__user-menu:last-child .menu > li > button {
+//   padding-inline-end: 0;
+// }
+
+// .cd-global-header__user-menu:last-child .menu > li > a::after,
+// .cd-global-header__user-menu:last-child .menu > li > button::after {
+//   display: none;
+// }
+
+// Last block in Global Header should have its dropdown right aligned */
+.cd-global-header .cd-global-header__user-menu:last-child .cd-global-header__dropdown {
+  right: 0;
+  left: initial;
+
+  [dir="rtl"] & {
+    right: initial;
+    left: 0;
+  }
+}
+
```

Comment:
ok I see that I overlooked this when doing the flexbox conversion in #358 

I think we can accomplish this with the same logical properties since flex keeps it flush in my other PR.

Reply:
Nevermind, this positions the dropdown and is a nice cosmetic to retain.